pkg/volume/plugins.go

```diff
@@ -173,3 +181,13 @@ func (pm *VolumePluginMgr) FindPluginByName(name string) (VolumePlugin, error) {
 	}
 	return pm.plugins[matches[0]], nil
 }
+
+// FindPluginByName fetches a plugin by name or by legacy name.  If no plugin
+// is found, returns error.
+func (pm *VolumePluginMgr) FindPersistentPluginByName(name string) (PersistentVolumePlugin, error) {
+	volumePlugin, err := pm.FindPluginByName(name)
+	if err != nil {
+		return nil, err
+	}
+	return volumePlugin.(PersistentVolumePlugin), nil
```

Comment:
This will panic if the returned plugin does not support persistentce.

Instead, try to convert it `pvp, ok := volumePlugin.(PersistentVolumePlugin)` and return an error if not ok.